src/components/members-list/index.js

```diff
@@ -2,19 +2,26 @@ import React from 'react';
 import MemberListItem from '@components/member-list-item';
 import classNames from '@components/members-list/member-list.module.scss';
 import { membersContext } from '@store/members/members-context';
+import { searchMemberContext } from '@store/search-members/searchMember-context';
+import { searchMembers } from '@helper-functions/search-members';
 
 const MembersList = () => {
   const {
     state: { membersArr },
   } = membersContext();
+  const { searchTerm } = searchMemberContext();
+  const filterMembers = searchMembers(membersArr, searchTerm);
   if (membersArr) {
     return (
       <div className={classNames.container}>
-        {membersArr.map((ele) => (
-          <React.Fragment key={ele.id}>
-            <MemberListItem memberDetails={ele} />
-          </React.Fragment>
-        ))}
+        {filterMembers.length > 0 &&
+          filterMembers.map((member) => (
+            <React.Fragment key={member.id}>
+              {(member.first_name || member.username) && (
```

Comment:
What is the significance of this condition? 
I believe first name and username are required fields in signup, open to other suggestions

Reply:
This code of this condition is previously embedded code not from my side.

Reply:
> What is the significance of this condition? I believe first name and username are required fields in signup, open to other suggestions

@harshith-venkatesh Thanks for this point. The condition is not getting used and it has no impact of its presence on the code. So I removed it now in my latest commit.